app/controllers/application_controller.rb

```diff
@@ -1,5 +1,9 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    @current_user = User.find(params[:user_id])
+  before_action :configure_permitted_parameters, if: :devise_controller?

```

Comment:
- [ ] While you have made a great effort in implementing the required changes, this has introduced a new bug in the codebase. I am currently unable to sign up. Upon submitting for a sign up I get the following error
![image](https://user-images.githubusercontent.com/81932478/206003709-aa292830-db5e-47f8-9729-238e5757490e.png)

Reply:
@Munsa1 Please excuse my recklessness 🙊, I've double-checked everything It should work fine now.